internal/endtoend/endtoend_test.go

```diff
@@ -1,3 +1,7 @@
+// Currently requires cgo for wasmtime and has line-ending issues on windows.
+//go:build cgo && !windows
```

Comment:
I tried a few things to normalize line endings in the test but ended up giving up since there were some non-trivial issues still